### ISSUE TITLE
Support for AngularJS 1.2.0 (tested with 1.2.0-rc.3)

### ISSUE THREAD

**Issue by kfiil** [mostr/angular-ui-multi-sortable#12]
First of all, thanks for the cool script! We are in the process of upgrade our Angular app to 1.2 and having some problems with the script. Everything works as usual but when we sort between two lists the sorting item will be shown twice on the receiving list but is only in the model once. Do you have any ideas for what causes this behavior?

**Comment by mostr** [comment]
Could you setup some jsfiddle or plunkr snippet with all the dependencies that cause the issue?

**Comment by kfiil** [comment]
Yes of cause, but accully it seems like the problems comes from angular-ui sortable, I spent a lot time trying to figure it out yesterday so all help is very welcome :smile: 

Plunkr
http://plnkr.co/edit/tiYWqxI3IsSYOhdTEuOH

**Comment by ghost** [comment]
I also use your script and have had the same issue since 1.2.0-rc3. The new stable didn't fix it snif. The strange thing is I don't have any problem with angular-ui, only on multi-sortable.

**Comment by mostr** [comment]
I hope to look at  this compatibility issues some time soon.

**Comment by kfiil** [comment]
That would be cool, to be fair i believe the problem is the combination of jQuery UI and AngularJS 1.2.0 and not angular-ui-multi-sortable. The main problem is, that jQuery UI updates the DOM and then when we update the model using $modelValue Angular also updates the DOM. See https://github.com/angular-ui/ui-sortable/pull/48 and https://github.com/angular-ui/ui-sortable/issues/47
